src/components/OntologyBrowser/SelectionTree.tsx

```diff
@@ -0,0 +1,139 @@
+import * as React from 'react';
+import { Component, Fragment, useState } from 'react';
+import { Input, Tag, Tree } from 'antd';
+import { TreeNode } from './store';
+
+import './SelectionTree.css';
+const { Search } = Input;
+
+type SelectionTreeProps = {
+  dataSource: TreeNode[];
+  checkedKeys: Array<string>;
+  onItemSelect: Function;
+  targetKeys: Array<string>;
+};
+
+type SelectionTreeState = {
+  treeData: TreeNode[];
+  expandedKeys: string[]
+};
+
+const getAllKeys = (data: TreeNode[], collectedKeys: string[] = [], counter = 1) => {
+  if (counter < 3) {
```

Comment:
magic number?

Reply:
yes its the number of level to expand. it was decided to be 3. But guess it could be in a constant.